### PR TITLE
[2.x] Update constraints

### PR DESCRIPTION
**Pull request by driesvints**
This PR:

- Drops support for Laravel 5.x versions
- Drops support for Symfony 3.x
- Requires a minimum of PHP 7.2
- Add support for Symfony 5.0

This will need to be a new major version release.

Might want to wait with merging this until https://github.com/bobthecow/psysh/pull/577 is merged and tagged.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -10,15 +10,18 @@
...
-        "illuminate/console": "~5.1|^6.0|^7.0",
-        "illuminate/contracts": "~5.1|^6.0|^7.0",
-        "illuminate/support": "~5.1|^6.0|^7.0",
-        "psy/psysh": "0.7.*|0.8.*|0.9.*",
-        "symfony/var-dumper": "~3.0|~4.0"
+        "php": "^7.2",
+        "illuminate/console": "^6.0|^7.0",
+        "illuminate/contracts": "^6.0|^7.0",
+        "illuminate/support": "^6.0|^7.0",
+        "psy/psysh": "^0.9.0",
```
Comment (GrahamCampbell) [inline composer.json]: `^0.9` is the same as `^0.9.0` and is more conventional.

Comment (driesvints) [reply composer.json]: Updated

**Comment by GrahamCampbell** [comment]
This seems a bit premature. It' possible (in fact, likely), that psysh 0.9.x will never support Symfony 5, only the next breaking version of psysh might.

**Comment by driesvints** [comment]
@GrahamCampbell how so? Doesn't seems like https://github.com/bobthecow/psysh/pull/577 contains any breaking changes?

**Comment by GrahamCampbell** [comment]
No, but it's been a year since the last release, and there are other changes that are likely to mean the next release will be 0.10.0.

**Comment by bobthecow** [comment]
Give me a heads up next time and I’ll cut a release ahead of Tinker’s 😛

The next PsySH release will be a “major” version (or whatever you call it because v0.x 😒) because there are several other breaking changes in the development branch already.

But it seems worth also adding a new v0.9.x release adding Symfony 5 support.

**Comment by bobthecow** [comment]
Ok! PsySH supports Symfony 5.0 as of v0.9.11, so this’ll work without any changes to your constraints.

**Comment by GrahamCampbell** [comment]
👍

**Comment by driesvints** [comment]
Thanks @bobthecow!

**Pull request merged**